src/components/CustomerQuota/ItemsSelection/Item.tsx

```diff
@@ -26,18 +27,34 @@ const isPodCampaign = (cartItemCategory: string): boolean =>
   cartItemCategory.includes("tt-token");
 
 /**
- * Check if pod is chargeable
+ * Check if pod is chargeable.
+ *
+ * Redemption of the first token is chargeable for passport customer.
  *
+ * @param id customerId
  * @param identifiers policy identifiers
  * @param cartItem policy cart item
  */
 const isPodChargeable = (
+  id: string,
   identifiers: PolicyIdentifier[],
   cartItem: CartItem
-): boolean =>
-  identifiers.length > 0 &&
-  cartItem.category.includes("tt-token-lost") &&
-  cartItem.descriptionAlert === "*chargeable";
+): boolean => {
+  if (identifiers.length > 0) {
+    if (cartItem.category.includes("tt-token-lost")) {
+      if (cartItem.descriptionAlert === "*chargeable") {
+        return true;
+      }
+    } else if (validatePassport(id)) {
+      if (cartItem.category.includes("tt-token")) {
+        if (cartItem.descriptionAlert === "*chargeable") {
```

Comment:
Is this required? Since if passport then it should be chargeable already

Reply:
o yea you're right!

Reply:
Added a follow up comment below, resolving this.